subm/src/discordGuild.js

```diff
@@ -64,6 +70,15 @@ function getContent(host, path, headers, { get }) {
  *   pending?: boolean,
  *   permissions?: string,
  * }} GuildMember
+ *
+ * https://discord.com/developers/docs/resources/user#user-object
+ * @typedef {{
+ *   id: Snowflake,
+ *   username: string,
+ *   discriminator: string,
+ *   avatar?: string,
+ *   email?: string, // ... etc.
```

Comment:
If we end up capturing email, looks like there is a `verified` flag for it.

Reply:
I punted on email from discord for now.